kotlinx-coroutines-core/common/test/flow/channels/FlowCallbackTest.kt

```diff
@@ -2,9 +2,9 @@
 
 package kotlinx.coroutines.flow
 
-import kotlinx.coroutines.testing.*
 import kotlinx.coroutines.*
 import kotlinx.coroutines.channels.*
+import kotlinx.coroutines.testing.*
```

Comment:
Unrelated: L1: Do we need this suppression here? I don't quite understand how it is relevant here.